src/smtsolvers/CoreSMTSolver.h

```diff
@@ -484,14 +485,13 @@ class CoreSMTSolver
     vec<lbool>         val_to_restore;             // For cancelUntilVarTemp
 
     //
-    // Proof production
+    // ResolutionProof production
     //
-    bool logsProofForInterpolation() const { return static_cast<bool>(proof); }
-    void finalizeProof(CRef finalConflict);
-    std::unique_ptr<Proof> proof;                 // (Pointer to) Proof store
+    void finalizeResolutionProof(CRef finalConflict);
+    std::unique_ptr<ResolutionProof> resolution_proof;                 // (Pointer to) ResolutionProof store
```

Comment:
I think we should use `resolutionProof` to be consistent with the style.

Reply:
I think that all member variables use snake case, only functions use camelCase, right?

Reply:
Now I am confused, e.g. in `MainSolver.h` both cases of naming member variables appear, even after applying the Clang formatting in #677

Reply:
All the new code has been using camelCase for everything for some years now.